package.json

```diff
@@ -3,19 +3,29 @@
   "version": "1.0.0",
   "description": "",
   "main": "x.js",
+  "type": "module",
   "scripts": {
-    "encrypt-secrets:triggers-oracle-node": "aws kms encrypt --key-id 'PUT_YOUR_AWS_KEY_ID_HERE' --plaintext fileb://./src/triggers-oracle-node/env-secrets.json --output text --query CiphertextBlob --output text | base64 -D > ./src/triggers-oracle-node/env-secrets-encrypted.json",
-    "encrypt-secrets:deposit-oracle-node": "aws kms encrypt --key-id 'PUT_YOUR_AWS_KEY_ID_HERE' --plaintext fileb://./src/deposit-oracle-node/env-secrets.json --output text --query CiphertextBlob --output text | base64 -D > ./src/deposit-oracle-node/env-secrets-encrypted.json",
-    "build:coin-market-cap-proxy": "rm -f ./build/coin-market-cap-proxy/build.zip && mkdir -p ./build/coin-market-cap-proxy && zip -j ./build/coin-market-cap-proxy/build.zip ./src/coin-market-cap-proxy/index.js",
-    "build:triggers-oracle-node": "rm -f ./build/triggers-oracle-node/build.zip && mkdir -p ./build/triggers-oracle-node && zip -j ./build/triggers-oracle-node/build.zip ./src/triggers-oracle-node/index.js ./src/triggers-oracle-node/oracle-smart-contract-abi.json ./src/triggers-oracle-node/env-secrets-encrypted.json",
-    "build:deposit-oracle-node": "rm -f ./build/deposit-oracle-node/build.zip && mkdir -p ./build/deposit-oracle-node && zip -j ./build/deposit-oracle-node/build.zip ./src/deposit-oracle-node/index.js ./src/deposit-oracle-node/oracle-smart-contract-abi.json ./src/deposit-oracle-node/env-secrets-encrypted.json"
+    "compile:triggers-oracle-node": "tsc --alwaysStrict -p ./tsconfig.triggers-oracle-node.json",
+    "compile:deposits-oracle-node": "tsc --alwaysStrict -p ./tsconfig.deposits-oracle-node.json",
+    "build:triggers-oracle-node": "node ./scripts/build.js 'triggers-oracle-node'",
+    "build:deposits-oracle-node": "node ./scripts/build.js 'deposits-oracle-node'",
+    "deploy:triggers-oracle-node": "node ./scripts/deploy.js './build/triggers-oracle-node.zip' 'goerli-flyweight-oracle-triggers'",
+    "deploy:deposits-oracle-node": "node ./scripts/deploy.js './build/deposits-oracle-node.zip' 'goerli-flyweight-oracle-deposits'",
```

Comment:
Just went with a hardhat-like scripts folder, for the build & release node scripts